Dockerfile.rhel7

```diff
@@ -0,0 +1,20 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.10 AS builder
+WORKDIR /go/src/github.com/openshift/secondary-scheduler-operator
+COPY . .
+
+RUN go build -o secondary-scheduler-operator ./cmd/cluster-kube-descheduler-operator
+
+FROM registry.ci.openshift.org/ocp/4.10:base
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+COPY --from=builder /go/src/github.com/openshift/secondary-scheduler-operator/secondary-scheduler-operator /
```

Comment:
How about we removing the ENTRYPOINT and making it consistent with Dockerfile??

```
COPY --from=builder /go/src/github.com/openshift/secondary-scheduler-operator/secondary-scheduler-operator /usr/bin/
```

Reply:
+1, let's stick with the form we have in https://github.com/openshift/cluster-kube-descheduler-operator/blob/master/Dockerfile.rhel7 to keep both files in sync.